docs/base-api.md

```diff
@@ -615,6 +624,7 @@ $baseApi->updateVideoLibrary(
         - `pip`
         - `airplay`
         - `fullscreen`
+    - To get a full list of possible value options for key `TranscribingCaptionLanguages`, see the [Get Languages](#get-languages) endpoint.
```

Comment:
Think this is the correct way to link to a section; the `Get Languages` will be added in #87